.circleci/config.yml

```diff
@@ -573,14 +573,12 @@ workflows:
                             - 1.27.2
                             - 1.27.3
                             - 1.27.4
+                            - 1.27.5
                         waf:
                             - "ON"
                             - "OFF"
                 name: build << matrix.nginx-version >> on << matrix.arch >> WAF << matrix.waf >>
             - build_openresty:
-                filters:
```

Comment:
Why did you remove that ? I think it is because the generate_jobs_yaml does not have these

Reply:
addressed in 440a0d312c7f2f069e7cf4fc5b705abc0e01ce5b